home.html

```diff
@@ -0,0 +1,314 @@
+                                <span class="material-icons ">star_outline</span>
+                                <span class="material-icons ">star_outline</span>
+                                <h5 class="small-padding">500</h5>
+                            </div>
+
+                        </div>
+                        <div class="org-price">
+                            <h4><strike>$100</strike></h4>
+                            <h4>60% Off</h4>
+                        </div>
+                        <div class="description">
+                            <P> Color : White</P>
+                            <P> Size : Large</P>
+                        </div>
+                        <div class="order-buttons-horizontal">
+                            <a href="#"><i class="fa fa-shopping-cart"></i>ADD TO CART</a>
+                            <a href="#">BUY NOW</a>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="card-horizontal shadow">
+                    <div class="image-container-horizontal">
+                        <img src="/Assets/shoe3.jpg" alt="adidas shoes">
+                    </div>
+                    <div class="content-container">
+                        <div class="heading">
+                            <h2>ADIDAS MAX</h2>
+                            <h4>The limited edition 2022</h4>
+                        </div>
+                        <div class="reduced-price">
+                            <h3>$40</h3>
+                            <div class="centered small-padding">
+                                <span class="material-icons selected">star</span>
+                                <span class="material-icons selected">star</span>
+                                <span class="material-icons selected">star</span>
+                                <span class="material-icons ">star_outline</span>
+                                <span class="material-icons ">star_outline</span>
+                                <h5 class="small-padding">500</h5>
+                            </div>
+
+                        </div>
+                        <div class="org-price">
+                            <h4><strike>$100</strike></h4>
+                            <h4>60% Off</h4>
+                        </div>
+                        <div class="description">
+                            <P> Color : White</P>
+                            <P> Size : Large</P>
+                        </div>
+                        <div class="order-buttons-horizontal">
+                            <a href="#"><i class="fa fa-shopping-cart"></i>ADD TO CART</a>
+                            <a href="#">BUY NOW</a>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+
+        </div>
+    </div>
+
+    <footer class="footer-container centered-column">
```

Comment:
Nice and clean code structure. No changes are required as of now.

Reply:
Thanks